bin/webdrivers/chrome/README.md

```diff
@@ -0,0 +1,20 @@
+# Chrome webdriver
+
+Chrome webdriver for Selenium Server Standalone, the browser automation framework.
+
+## Version
+Release numbers are synchronised with the Selenium versions.
+Version of this release is *v2.21.0* 
+
+## Installation
+
+#### Inside your robofile
+```
+-Dwebdriver.chrome.driver=.\\vendor\\joomla-projects\\selenium-server-standalone\\bin\\webdrivers\chrome\chromedriver.exe
```

Comment:
Excellent job! 

mmm I guess being .exe these are just for Windows. I think we would need the executable for linux/macos systems too